app/views/users/webauthn_setup/new.html.erb

```diff
@@ -33,20 +38,38 @@
 
   <%= hidden_field_tag :platform_authenticator, @platform_authenticator, id: 'platform_authenticator' %>
   <% if !@platform_authenticator %>
-    
-      <%= render ValidatedFieldComponent.new(
-            form: f,
-            name: :name,
-            required: true,
-            label: @presenter.nickname_label,
-            hint: @presenter.device_nickname_hint,
-            input_html: {
-              id: 'nickname',
-              class: 'font-family-mono',
-              size: 16,
-              maxlength: 20,
-            },
-          ) %>
+    <fieldset aria-labelledby="webauthn-intro" class="padding-0 border-0 margin-y-4 margin-x-0">
+      <%= render ProcessListComponent.new(connected: true) do |c| %>
+        <%= c.with_item(heading: t('forms.webauthn_setup.step_1'), heading_id: 'step-1-label') do %>
+          <p><%= t('forms.webauthn_setup.step_1a') %></p>
+          <%= render ValidatedFieldComponent.new(
+                form: f,
+                name: :name,
+                required: true,
+                label: false,
+                hint: @presenter.device_nickname_hint,
+                wrapper_html: { class: 'margin-bottom-0' },
+                input_html: {
+                  aria: { labelledby: 'step-1-label' },
+                  id: 'nickname',
+                  class: 'font-family-mono',
+                  size: 16,
+                  maxlength: 20,
+                },
+              ) %>
+        <% end %>
+        <%= c.with_item(heading: t('forms.webauthn_setup.step_2')) do %>
+            <% if @mobile %>
+              <%= image_tag asset_url('mfa-options/security_key_mobile.gif'), width: 420, height: 193, class: 'rounded-xl', alt: t('forms.webauthn_setup.step_2_image_mobile_alt') %>
```

Comment:
With the dimensions attributes, one thing I'm noticing is that the image becomes distorted at small sizes because we also have a `max-width: 100%` style for images to prevent overflow. It happens for most standard phone sizes, but especially noticeable on smaller screens like iPhone SE.

The article I linked earlier includes a [section about dealing with this](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/#how-css-interacts-with-element-widths-and-heights) and I think it'd be reasonably safe to apply that style to every image, but we could also limit it to this image or maybe even use a utility helper like `height-auto`.

<img src="https://github.com/18F/identity-idp/assets/1779930/1cda343d-0acd-4a9f-b930-79a054bd6edb" width="350" alt="screenshot">

Reply:
Added the class `height-auto` to both animated images. In a quick initial test on a mobile device the change looks better.